edx_notifications/server/web/static/edx_notifications/js/test/specs/counter_icon_view_spec.js

```diff
@@ -0,0 +1,30 @@
+describe("CounterIconView", function(){
+
+
+    beforeEach(function(){
+        jasmine.Ajax.install();
+    });
+
+    afterEach(function() {
+      jasmine.Ajax.uninstall();
+    });
+
+    it("counter view model url is set successfully", function(){
+         $('body').append('<div id="edx_notification_container" />');
+        this.counter_view = new CounterIconView({
```

Comment:
I would move the construction of the view into a beforeEach method, so that future tests can share it. It also has the benefit of making the test shorter and easier to understand.